types/leaflet/index.d.ts

```diff
@@ -626,7 +626,7 @@ export interface PathOptions extends InteractiveLayerOptions {
     opacity?: number;
     lineCap?: LineCapShape;
     lineJoin?: LineJoinShape;
-    dashArray?: string;
+    dashArray?: string | number[];
```

Comment:
I had a quick look at the underlying library and it didn't obviously support non-string values for `dashArray`.  Do you have a link to the code that's consuming an array instead?

Reply:
I'm looking at this PR https://github.com/Leaflet/Leaflet/pull/6200, which is part of Leaflet 1.3.2. The documentation does not appear to be updated.

Reply:
Perfect.  Thanks!